app/models/automation_request.rb

```diff
@@ -75,4 +74,16 @@ def self.build_attrs(parameters, user)
     end
   end
   private_class_method :build_attrs
+
+  def self.parse_schedule_options(parameters)
+    {:schedule_type => "schedule"}.tap do |k, _v|
+      if parameters['schedule_time']
+        k[:schedule_time] = Time.zone.parse(parameters['schedule_time'])
```

Comment:
I thought I clicked add comment...

Anyway, can we safely drop the  "5" for 5 days from now use case?  Is this stored anywhere in the DB or only user input that we can WARN about the new format?

Reply:
it's not exposed to the UI and, even better, it's only a thing we stuff into the hash to do disgusting parsing on later. it's not saved until after it gets converted to a datetime when it gets queued.